docs/outlook/internet-headers.md

```diff
@@ -1,15 +1,19 @@
 ---
 title: Get and set internet headers
 description: How to get and set internet headers on a message in an Outlook add-in.
-ms.date: 11/04/2019
+ms.date: 04/10/2019
 localization_priority: Normal
 ---
 
 # Get and set internet headers on a message in an Outlook add-in
 
```

Comment:
Alternatively, you can add an introductory sentence then insert the image after that.
```suggestion
The ability to set custom headers on an email then read them from the recipient's side enables metadata to be shared between users.

![Diagram of internet headers. Text: User 1 sends email. Add-in manages custom internet headers while user is composing email. User 2 receives the email. Add-in gets internet headers from received email then parses and uses custom headers. ](../images/outlook-internet-headers.png)

```